packages/node_modules/@ciscospark/react-component-utils/src/uuid-utils.test.js

```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
+import {buildHydraRoomId, hydraTypes} from '@webex/common';
+
+import {validateAndDecodeId} from './uuid-utils';
+
+describe('uuid-utils', () => {
+  describe('validateAndDecodeId', () => {
+    it('returns an id object for a uuid input', () => {
+      const ID = uuid.v4();
+
+      expect(validateAndDecodeId(ID)).toEqual({id: ID});
+    });
+
+    it('deconstructs a hydra ID', () => {
+      const id = uuid.v4();
+      const cluster = 'us';
+      const hydraId = buildHydraRoomId(id, cluster);
+
+      expect(validateAndDecodeId(hydraId)).toEqual({id, cluster, type: hydraTypes.ROOM});
+    });
+
+    [null, '', 'AB%CD']
```

Comment:
Could you separate these into three `it` blocks? We forgot to mention it during our scrum meetings, but I believe we didn't want the test looping (am I correct @lalli-flores?).